### PR TITLE
Fetch cases.json, deaths.json, and projections.json data files from API

### PR DESCRIPTION
**Pull request by shaileshmahajan**
This JIRA addresses the issue - Fetch cases.json, deaths.json, and projections.json data files from API
Fixes - http://jira/browse/MRIID-75

Testing Guidelines - 
- Pull down the branch.
- Update the .env to add `REACT_APP_BACKEND_URL=https://gtwn1uelqb.execute-api.us-east-1.amazonaws.com/dev/api/`
- Start the server
- Check the network tab and make sure that JSON data is returned by API call (as oppose to reading local JSON files)
- Click around the entire UI and make sure its functional (especially when Covid 19 dropdown and its sub-elements are selected)

### CODE REVIEW

File: env.sample [hunk 1]
```diff
@@ -1 +1,2 @@
 REACT_APP_MAPBOX_ACCESS_TOKEN='pk.eyJ1IjoiY29tcGVwaSIsImEiOiJjam4zaDd0cm4wZWE1M3JsYm8zMnJtcTU2In0.my4lxowcqSdygJmxVgz5sA'
+REACT_APP_BACKEND_URL=
```
Comment (fernandodonaire) [inline env.sample:2]: Could you please add the api url to the `env.sample`? Or is there some reason we wouldn't want to have that here?

**Review by fernandodonaire — APPROVED** [review]
This is working great @shaileshmahajan !!! Great job! 

Since it takes a bit of time to fetch the covid data from the API, we probably want to add some loading message/spinner to the chart and map components just in case a user quickly changes to the Covid view before the data is fetched. But we can do that in another ticket(s) 😄

File: env.sample (hunk 1 above)
Comment (shaileshmahajan) [reply env.sample]: @fernandodonaire . Done!!!

**Comment by shaileshmahajan** [comment]
> This is working great @shaileshmahajan !!! Great job!
> 
> Since it takes a bit of time to fetch the covid data from the API, we probably want to add some loading message/spinner to the chart and map components just in case a user quickly changes to the Covid view before the data is fetched. But we can do that in another ticket(s) 😄

@fernandodonaire Looks like you beat me to it. I see MRIID-84

**Pull request merged**